pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = ["click", "pyyaml", "xmltodict", "rdflib", "beartype", "uritools"
 dynamic = ["version"]
 
 [project.optional-dependencies]
-test = ["pytest", "pyshacl"]
+test = ["pytest", "pyshacl", "tzdata"]
```

Comment:
just a test dependency? I would have expected changes in non-test dependencies.

Reply:
As I understand things, the library itself doesn't care about any timezone besides UTC (which is included in `datetime`).  I only need it for testing because I need to create a localized datetime object in the tests to check that we generate a correct ISO 8601 date string from it.